src/components/layout.tsx

```diff
@@ -45,7 +45,7 @@ const Content = styled.article<{ fullWidth?: boolean; wide?: boolean }>`
       : 'max-width: 748px;'}
   flex: 1;
   position: relative;
-  z-index: 100;
+  z-index: 1001;
```

Comment:
Mentioned in Slack: I think this PR might have broken the search on the website/docs, see screenshot (changing the z-index back to old value fixes it) 
<img width="1200" alt="Screenshot 2023-06-07 at 14 33 58" src="https://github.com/prisma/docs/assets/1328733/e73cb150-2858-4b77-9750-eafa6b700315">